userguide/services.rst

```diff
@@ -238,13 +238,8 @@ Licensed Asigra Backup software can use %brand% as the storage backend.
 For the initial backend configuration, click
 :menuselection:`Services --> Asigra DS-System`. When prompted to choose
 the :guilabel:`Base Filesystem`, select the dataset to store the Asigra
-backups then click :guilabel:`OK`. The system will automatically create
-the required database entries and start the service.
-
-While the service is running, the :guilabel:`Open DS-Operator Web Interface`
-button appears in :menuselection:`Services --> Asigra DS-System`. Click
-:guilabel:`Open DS-Operator Web Interface` to download and launch the
-Asigra management application.
+backups then click :guilabel:`OK`. The system automatically creates
```

Comment:
Take a breath here, and "the system" is ambiguous.  Fortunately, it is not needed:
```suggestion
backups, then click :guilabel:`OK`. Any required database entries are created and the service is started.
```